irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -104,7 +107,10 @@ namespace iroha {
 
       return expected::makeValue<std::unique_ptr<TemporaryWsv>>(
           std::make_unique<TemporaryWsvImpl>(
-              std::move(sql), factory_, perm_converter_));
+              std::move(sql),
+              factory_,
+              perm_converter_,
+              log_manager_->getChild("TemporaryWorldStateView")));
```

Comment:
Seems, that all `getChild` invocations should be reworked with another approach.